Neos.Flow/Classes/ObjectManagement/DependencyInjection/ProxyClassBuilder.php

```diff
@@ -312,7 +318,7 @@ protected function buildConstructorInjectionCode(Configuration $objectConfigurat
             $index++;
         }
 
-        return $code;
+        return $doBuildCode ? $code : '';
```

Comment:
The actual building of `$code` could be skipped, if the check was done after the `foreach ($argumentConfigurations as  …` loop…

Reply:
can be done later, merging to move forward :)